utils/challenges/challenges.js

```diff
@@ -0,0 +1,36 @@
+import firebase from "../firebase/firebase";
+
+export async function setCurrentChallenge(teamId) {
+  // pick a random challenge
+  const challengeCountSnapshot = await firebase
+    .database()
+    .ref("/challenges/count")
+    .once("value");
+  const challengeCount = challengeCountSnapshot.val();
+  // challenge ids are indexed at 0
+  const challengeId = Math.floor(Math.random() * challengeCount);
+
+  const assignedTime = firebase.database.ServerValue.TIMESTAMP;
+  // time since the Unix epoch, in milliseconds) as determined by the Firebase servers
+  // see https://firebase.google.com/docs/reference/js/firebase.database.ServerValue
+
+  const newChallenge = {
+    challengeId,
+    teamId,
+    assignedTime
```

Comment:
Not sure if necessary, but consider adding a `completedTime` field to this object?

Reply:
Schema on firebase is flexible so there's actually a few missing fields like submissions as well.  It can be added later on after creation if we decide we need it.